src/implementations/twig/components/select/select.html.twig

```diff
@@ -113,6 +118,12 @@
 
   {% if _invalid and _invalid_text is not empty %}
     <div class="ecl-feedback-message">
+      {% if _invalid_icon is not empty %}
```

Comment:
This is the reason why we have the empty line in the test.. :) We need to use dashes `-` in this case to avoid it.